### PR TITLE
Update generations to support netstandard

### PR DESCRIPTION
**Pull request by kartang**
1) dnxcore50 is updated to netstandardapp1.5-> In reference to the following links:
https://github.com/NuGet/Home/issues/1956

### CODE REVIEW

**Comment by kartang** [comment]
/ @davidfowl

**Comment by upendras** [comment]
:shipit:

File: src/Core/NuGet/Core.NuGet-Beta/Package.nuspec [hunk 1]
```diff
@@ -24,7 +24,7 @@
       </group>
       <group targetFramework="net46">
       </group>
-      <group targetFramework="dnxcore50">
+      <group targetFramework=".NETStandardApp,Version=v1.5">
         <dependency id="System.Collections" version="4.0.10-beta-22816" />
```
Comment (SergeyKanzhelev) [inline src/Core/NuGet/Core.NuGet-Beta/Package.nuspec]: As we shipping stable version of NuGet all references should be changed to the latest stable version of dependencies. No betas.

File: src/Core/NuGet/Core.NuGet-Beta/Package.nuspec [hunk 2]
```diff
@@ -24,7 +24,7 @@
       </group>
       <group targetFramework="net46">
       </group>
-      <group targetFramework="dnxcore50">
+      <group targetFramework=".NETStandardApp,Version=v1.5">
```
Comment (SergeyKanzhelev) [inline src/Core/NuGet/Core.NuGet-Beta/Package.nuspec]: Why `netstandardapp`, not `netstandard`? It should work on any platform and cannot be run by itself.

File: src/Core/NuGet/Core.NuGet-Beta/Package.nuspec (hunk 1 above)
Comment (kartang) [inline src/Core/NuGet/Core.NuGet-Beta/Package.nuspec]: Will change that. Thanks!

Comment (SergeyKanzhelev) [inline src/Core/NuGet/Core.NuGet-Beta/Package.nuspec]: Also as we making it stable - remove this definition from repo: https://github.com/Microsoft/ApplicationInsights-dotnet/tree/master/src/Core/NuGet/Core

File: src/Core/NuGet/Core/Package.nuspec [hunk 3]
```diff
@@ -24,14 +24,40 @@
       </group>
       <group targetFramework="net46">
       </group>
+      <group targetFramework=".NETStandard,Version=v1.5">
```
Comment (SergeyKanzhelev) [inline src/Core/NuGet/Core/Package.nuspec]: So you changed to NetStandard now. @davidfowl - this is the main question. Should we use netstandard or netstandardAPP here?

**Comment by SergeyKanzhelev** [comment]
:shipit:

**Pull request merged**